modules/nw-kube-proxy-config.adoc

```diff
@@ -1,13 +1,16 @@
 // Module included in the following assemblies:
-// * networking/configuring-kubeproxy.adoc
+//
+// * networking/openshift_sdn/configuring-kube-proxy.adoc
 
 [id="nw-kube-proxy-config_{context}"]
 = kube-proxy configuration parameters
 
-You can modify the following `kubeProxyConfig` parameters:
+You can modify the following `kubeProxyConfig` parameters.
+
+IMPORTANT: Because of performance improvements introduced in {product-title} 4.3 and greater, adjusting the `iptablesSyncPeriod` parameter is not recommended.
```

Comment:
@danwinship -- I added a note here about `iptablesSyncPeriod`.

Reply:
"not recommended" makes it sound like changing the parameter will have an effect, but we don't want them to do it anyway. But in fact, changing the parameter will not have any effect at all. Maybe "not necessary" instead of "not recommended"? (Or "no longer necessary".)

likewise in the other sections